static/css/components/page-banner.less

```diff
@@ -3,16 +3,22 @@
  * https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library#pagebanner
  */
 @import "components/buttonBtn.less";
+@import "less/breakpoints.less";
 
 .page-banner {
+  display:none;
   color: @white;
-  font-family: @lucida_sans_serif-1;
+  font-family: "Lucida Grande", Verdana, Geneva, Helvetica, Arial, sans-serif;
```

Comment:
You'll want to undo this comment

Reply:
in the new commit i kept 9 and discarded 11, should i include both?